sys/kern/signal.c

```diff
@@ -445,27 +445,47 @@ void sig_onexec(proc_t *p) {
   }
 }
 
+bool sig_should_stop(sigaction_t *sigactions, signo_t sig) {
+  return (sigactions[sig].sa_handler == SIG_DFL && defact(sig) == SA_STOP);
+}
```

Comment:
Is it used outside of `signal.c`?
If not maybe changing it into static function will be a good idea?

Reply:
Done.